src/Artsy/Analytics/Schema/Values.ts

```diff
@@ -45,6 +46,8 @@ export enum ActionType {
    */
   ClickedBuyNow = "Clicked buy now",
   ClickedMakeOffer = "Clicked make offer",
+  ClickedContactGallery = 'Clicked "Contact Gallery"',
```

Comment:
Does this need to be in "" ? seems like the other ones are just plain strings and I think at some point they get converted to the `_` versions. I think it's cleaner not to have double quotes in the names.

Reply:
I'm just taking it directly from the sheet! I think I got the OK for it.

Reply:
Then I should change the events above to include the "" as well. Strange to have it both ways one after the other.

Reply:
But to be a stickler - we don't even capitalize works on the button any more. So it should probably be 'Clicked "Contact gallery"' after all.